config/buildbuddy.local.yaml

```diff
@@ -2,6 +2,7 @@ app:
   log_error_stack_traces: true
   streaming_http_enabled: true
   invocation_log_streaming_enabled: true
+  community_link_enabed: false
```

Comment:
revert?

Reply:
ah damn I forgot to revert after my local test